### PR TITLE
Update resources.md

### PR DESCRIPTION
**Pull request by Doishy**
Inspired by that reddit post to update a few bits.

### CODE REVIEW

File: markdown/appendices/resources.md [hunk 1]
```diff
@@ -4,11 +4,14 @@ title: Other Doomsday Resources
 
 ## Streams
 
+You can often find people streaming in the discord server. Alternatively check out these channels:
```
Comment (angrybacon) [inline markdown/appendices/resources.md:7]:
```suggestion
You can often find people streaming in the Discord server. Alternatively check
out these channels:
```

File: markdown/appendices/resources.md [hunk 2]
```diff
@@ -4,11 +4,14 @@ title: Other Doomsday Resources
...
+
 - Lee *d8dk32* Curtis:
   [Twitch](https://www.twitch.tv/ddftguy)
   [YouTube](https://www.youtube.com/channel/UCD0Os6qvXicEZl6gJ_xPXGw)
 - Brandon *emidln* Adams:
   [Twitch](https://www.twitch.tv/adam4300)
+  [Youtube](https://www.youtube.com/channel/UCaUTfix3JtYYwpLHBSFunIw)
```
Comment (angrybacon) [inline markdown/appendices/resources.md:14]:
```suggestion
  [YouTube](https://www.youtube.com/channel/UCaUTfix3JtYYwpLHBSFunIw)
```

File: markdown/appendices/resources.md [hunk 3]
```diff
@@ -35,17 +53,22 @@ title: Other Doomsday Resources
 
 ## Videos
 
+- [LeFly runs a League with Meandeck (February 2021) - German](https://www.youtube.com/watch?v=pkpixIywm1w)
+- [Andreas Mengucci runs a league with UB Meandeck (December 2020)](https://www.youtube.com/watch?v=8HlIvDf7iDg)
+- [Anuraag Das runs some ANT/Doomsday with Ki Sawatari (December 2020)](https://www.youtube.com/watch?v=u-kyFx2VJ0c)
 - [Rodrigo *Sharkcaster_Mage* Togores runs a set of leagues with BUrg Meandeck (September 2020)]
   (https://www.youtube.com/playlist?list=PLowHzeuy7CRM8N34LB8Oc0XGCLIxB3Q3z)
-- [TwinlessTwinMTG runs a league with Monkeyscantcry]
+- [TwinlessTwinMTG runs a league with Monkeyscantcry (Expired)]
```
Comment (angrybacon) [inline markdown/appendices/resources.md:61]: Do we want to keep those? Since they're not available anymore

**Review by angrybacon — CHANGES_REQUESTED** [review]
Might also want to fix the wrong links that you spotted ie. join all links that are split over 2 lines into a single line

I can do it later though

File: markdown/appendices/resources.md (hunk 3 above)
Comment (Doishy) [reply markdown/appendices/resources.md:61]: Probably not. Which is a shame because they were good ¬.¬
Bloody people not saving vods properly.

**Review by angrybacon — APPROVED** [review]

**Pull request merged**